tinyms/serving/servable/servable.py

```diff
@@ -20,7 +20,15 @@
 from tinyms import model
 from tinyms.utils.predict.predict import cyclegan_predict
 
-servable_path = '/etc/tinyms/serving/servable.json'
+server_path = '/etc/tinyms/serving/'
+
+if os.path.exists('temp.json'):
```

Comment:
I don't know why should add `temp.json` file assert here

Reply:
I will create a temporary file to store the specified path when the user specifies serving_path, but if the user does not specify it, then the temporary file will not exist. So I need to make a judgment here whether I use the specified path in the file or the default path